src/validation.cpp

```diff
@@ -3435,6 +3435,12 @@ bool ProcessNewBlockHeaders(const std::vector<CBlockHeader>& headers, CValidatio
         }
     }
     NotifyHeaderTip();
+    {
+        LOCK(cs_main);
+        if (IsInitialBlockDownload() && ppindex && *ppindex && pindexBestHeader) {
+            LogPrintf("Synchronizing blockheaders, height: %d (~%.2f%%)\n", (*ppindex)->nHeight, 100.0/((*ppindex)->nHeight+(GetAdjustedTime() - (*ppindex)->GetBlockTime()) / Params().GetConsensus().nPowTargetSpacing) * (*ppindex)->nHeight);
```

Comment:
```suggestion
            static int last_logged_block_header_height = 0;
            if (last_logged_block_header_height == 0 || (*ppindex)->nHeight - last_logged_block_header_height >= 100000) {
                LogPrintf("Synchronizing block headers: height=%d (~%.2f%%)\n", (*ppindex)->nHeight, 100.0/((*ppindex)->nHeight+(GetAdjustedTime() - (*ppindex)->GetBlockTime()) / Params().GetConsensus().nPowTargetSpacing) * (*ppindex)->nHeight);
                last_logged_block_header_height = (*ppindex)->nHeight;
            }
```

Reply:
Why?

Reply:
@promag See previous discussion in this PR.

Reply:
These are logs and come at a good pace, I'd merge this as it is.